dist/pell.css

```diff
@@ -26,3 +26,20 @@
   height: 30px;
```

Comment:
@hgm1284 Thanks for your changes 👍 , but you must put them in the `src/pell.scss` and run Gulp to respect the development pipeline that is used by @jaredreich